assets/rich_text_components_definitions.js

```diff
@@ -45,7 +46,8 @@ var richTextComponents = {
     "frontend_id": "image",
     "tooltip": "Insert image",
     "icon_data_url": "/rich_text_components/Image/Image.png",
-    "preview_url_template": "/imagehandler/<[explorationId]>/<[filepath.name]>",
+    "preview_url_template_dev": "/imagehandler/<[explorationId]>/<[filepath.name]>",
+    "preview_url_template_production": "/<[urlWithBucket]>/<[explorationId]>/<[filepath.name]>",
```

Comment:
Don't have `urlWithBucket`, Have `"https://storage.googleapis.com/<[bucketName]>/<[explorationId]>/<[filepath.name]>"` here and later interpolate just the bucket name.

Reply:
Done!